### PR TITLE
Update pull request pipeline

### PR DESCRIPTION
**Pull request by eleanorreem**
Rename the pipeline and remove staging from the list of branches to run this on

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 
 on:
   push:
-    branches: [develop, staging, main]
+    branches: [develop, main]
   pull_request:
-    branches: [develop, staging, main]
+    branches: [develop, main]
 
 jobs:
```
[the remark this answers is not in the document]
Comment (swetha-charles) [reply .github/workflows/pull-request.yml:9]: There are three tests at the moment

File: .github/workflows/pull-request.yml [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
   build:
```
Comment (swetha-charles) [inline .github/workflows/pull-request.yml:10]: What about renaming this from "build" to "lint-build-test" (if we include tests). 

The other option is to pull out "lint" and "test" as separate jobs. I don't feel that this is worth the effort at the moment imo

File: .github/workflows/pull-request.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (eleanorreem) [reply .github/workflows/pull-request.yml:9]: I ran yarn test and nothing happened so assumed there weren't any

**Review by swetha-charles — DISMISSED** [review]
Noice

File: .github/workflows/pull-request.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (swetha-charles) [reply .github/workflows/pull-request.yml:9]: My bad! That was for backend. Front end has 🕺 no tests 💃

**Pull request merged**